README.md

```diff
@@ -66,6 +66,14 @@ In addition, it will recursively discover configuration files in `beamium.d` dir
 
 Furthermore, Beamium support multiple formats for configuration files which are `hjson`, `json`, `toml`, `yaml`, `yml` or `ini`.
 
+
+Also, Beamium can be started with several labels put as env vars, they must be prefixed by `BEAMIUM_LABEL`.
+
+Ex:
+```sh
+BEAMIUM_LABEL_HOST=myhost ./beamium -v
```

Comment:
You spoke about some others behaviours, could you write it there too?

Reply:
the 2nd one is described here https://github.com/ovh/beamium/pull/119/files#diff-04c6e90faac2675aa89e2176d2eec7d8R90